.github/workflows/spell-check.yaml

```diff
@@ -29,8 +29,7 @@ jobs:
           checkout: true
           post_comment: 0
           dictionary_source_prefixes: '{"mondoo": "https://raw.githubusercontent.com/mondoohq/spellcheck-dictionary/main/", "cspell": "https://raw.githubusercontent.com/check-spelling/cspell-dicts/v20230509/dictionaries/"}'
-          extra_dictionaries:
-            cspell:aws/aws.txt
+          extra_dictionaries: cspell:aws/aws.txt
```

Comment:
shouldnt this be on a new line?

Reply:
from yaml perspective this does not make a difference